pkg/cloud/azure/assets/cloud-controller-manager-deployment.yaml

```diff
@@ -94,7 +141,7 @@ spec:
                 --leader-elect-retry-period=26s \
                 --leader-elect-resource-namespace=openshift-cloud-controller-manager
           volumeMounts:
-            - name: host-etc-kube
+            - name: merged-cloud-config
               mountPath: /etc/kubernetes
               readOnly: true
             - name: config-accm
```

Comment:
The pod is mounting and reading the cloud.conf from here rather than from the `/etc/kubernetes` location.
You need to change this mount to be the merged cloud config rather than the `/etc/kubernetes` mount